RegisterBackend.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
This PHP should be inside of the API folder in `index.php` to be run with an API call.